app/views/students/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1><%= @student.name %></h1>
-<h2><%= @student.house %></h2>
+<h2><%= @student.house.name %></h2>
 <div class="student">
-  <%= image_tag @student.img_url />
+  <%= image_tag "#{@student.img_url}" %>
```

Comment:
Don't think you need to use string interpolation here since `@student.img_url` is a string.